src/handlers/completion.rs

```diff
@@ -284,32 +421,46 @@ async fn find_completions(
     })
 }
 
-fn new_string_arg_completion(value: String) -> CompletionItem {
+fn new_string_arg_completion(
+    value: String,
+    trigger: Option<String>,
+) -> CompletionItem {
+    let trigger = trigger.unwrap_or_else(|| "".to_string());
```

Comment:
could you help me me understand what is happening here?

Reply:
basically just says if trigger coming from the params is `None` then default it to an empty string